### ISSUE TITLE
[FEATURE] Update wash to use pre-built assets from releases

### ISSUE THREAD

**Issue by lachieh** [wasmCloud/wasmCloud#875]
Once #874 is complete, we should update wash so that when running `wash ui` it will:
- check for local assets and download if none exist
- check for updated release and download if newer than local
- set specific release with option like `wash ui --version <washboard release>`

**Comment by connorsmith256** [comment]
@lachieh I propose we remove the second bullet point from the requirements here and file as a separate nice-to-have

**Comment by lachieh** [comment]
Was just going to ask about that—agreed. Want to split it out into a new issue as a `good-first-issue`?

**Comment by lachieh** [comment]
No, nevermind, let's add it to #848
